### ISSUE TITLE
Installation failed with node 4.2.1

### ISSUE THREAD

**Issue by fernandofranca** [davidmarkclements/cute-stack#10]
Errors from `npm-debug.log`

```
795 error Linux 4.0.9-boot2docker
796 error argv "/usr/local/bin/node" "/usr/local/bin/npm" "install" "cute-stack"
797 error node v4.2.1
798 error npm  v2.14.7
799 error code ELIFECYCLE
800 error v8-flags@0.1.2 preinstall: `node ./scripts/preinstall`
800 error Exit status 1
801 error Failed at the v8-flags@0.1.2 preinstall script 'node ./scripts/preinstall'.
801 error This is most likely a problem with the v8-flags package,
801 error not with npm itself.
801 error Tell the author that this fails on your system:
801 error     node ./scripts/preinstall
801 error     npm owner ls v8-flags
801 error There is likely additional logging output above.
```

**Comment by roblav96** [comment]
same :(

**Comment by roblav96** [comment]
npm install --save cute-stack@1.2

**Comment by davidmarkclements** [comment]
@roblav96 - this is fixed with 1.4 -  it took a long time to get around, sorry about that - if you get a chance to test spin it do let me know if there's still any problems your end

**Comment by roblav96** [comment]
ill give it a shot when i get home. thanks!
